gazelle/python/testdata/directive_python_generate_proto/test7_removes_when_unnecessary/BUILD.out

```diff
@@ -0,0 +1 @@
+# gazelle:python_generate_proto true
```

Comment:
Add test case: "removes just the `py_proto_library` when `python_generate_proto=false` and a `proto_library` exists"